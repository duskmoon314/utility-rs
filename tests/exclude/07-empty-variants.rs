use utility_types::Exclude;

#[derive(Debug, PartialEq, Exclude)]
#[exclude(arg(
    ident = Terrestrial,
    variants(),
    derive(Debug, PartialEq, Clone, Copy)
))]
pub enum Planet {
    Mercury,
    Venus,
    Earth,
    Mars,
    Jupiter,
    Saturn,
    Uranus,
    Neptune,
}

fn main() {}
