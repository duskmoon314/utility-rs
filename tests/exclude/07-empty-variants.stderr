error: Too few items: Expected at least 1
 --> tests/exclude/07-empty-variants.rs:6:5
  |
6 |     variants(),
  |     ^^^^^^^^^^
